admin/package-files/docker-plugin/upstart/flocker-docker-plugin.conf

```diff
@@ -0,0 +1,11 @@
+# flocker-docker-plugin - flocker-docker-plugin job file
+
+description "Flocker Docker Plugin"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on starting docker
```

Comment:
:question: Not sure if it's possible / desirable, but the plugin doesn't start automatically after installing the package.

```
root@acceptance-test-richardw-0:~# aptitude install clusterhq-flocker-docker-plugin
The following NEW packages will be installed:
  clusterhq-flocker-docker-plugin 
...
Preparing to unpack .../clusterhq-flocker-docker-plugin_1.2.0-0.dev.1.152.g94a1d7d_all.deb ...
Unpacking clusterhq-flocker-docker-plugin (1.2.0-0.dev.1.152.g94a1d7d) ...
Processing triggers for ureadahead (0.100.0-16) ...
Setting up clusterhq-flocker-docker-plugin (1.2.0-0.dev.1.152.g94a1d7d) ...
Processing triggers for ureadahead (0.100.0-16) ...
```

```
root@acceptance-test-richardw-0:~# initctl list | grep flocker
flocker-container-agent start/running, process 25085
flocker-dataset-agent start/running, process 25074
flocker-control start/running, process 25037
flocker-docker-plugin stop/waiting

```